webviz_ert/controllers/response_correlation_controller.py

```diff
@@ -490,6 +490,31 @@ def update_corr_param_resp(
         parameters: List[str],
         corr_param_resp: Dict,
     ) -> Optional[Dict]:
+        """
+        update_corr_param_resp is a callback function responsible for handling
+        which response and parameter are active.
+
+        The correlation plots display correlation between a given response and
+        paramater (at an index, see `corr_xindex`) for a chosen ensemble. Hence
+        the need to keep track of which of potentially many selected responses
+        and parameters, respectively, is currently active.
```

Comment:
"are" currently active

Reply:
thanks : ) i was puzzling over that one when i wrote it, i really wish i could express myself in a simpler way...

Reply:
done